pxr/usdImaging/usdImaging/delegate.cpp

```diff
@@ -2335,8 +2335,9 @@ UsdImagingDelegate::Get(SdfPath const& id, TfToken const& key)
             // XXX(UsdImaging): We use cachePath directly as usdPath here,
             // but should do the proper transformation.  Maybe we can use
             // the primInfo.usdPrim?
-            TF_VERIFY(_GetUsdPrim(cachePath).GetAttribute(key)
-                      .Get(&value, _time),
+            UsdPrim prim = _GetUsdPrim(cachePath);
+            UsdAttribute attr = prim ? prim.GetAttribute(key) : UsdAttribute();
+            TF_VERIFY(attr && attr.Get(&value, _time),
```

Comment:
Hey @marktucker !  I think the only concern we might have here is performance fallout.  One thing I am fairly confident about is that if the prim is valid, there should be no way that `prim.GetAttribute("mumble").Get()` would ever crash.  So I think you can get away with something like:
`TF_VERIFY(prim && prim.GetAttribute(key).Get(&value, _time))` which imposes only one extra validity check.